docs/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin-migration.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Plugin migration guide for v4 - Strapi Developer Docs
+description: Migrate your plugins from v3.6.8 to v4.0 with step-by-step instructions
```

Comment:
Assuming this will be v3.6.9 and not v3.6.8 but we can adjust depending on the next release cycle

Reply:
Yep, actually we thought the release of v3.6.9 (most about the password update) was planned for last Wednesday so I went a bit further. Will change 3.6.9 occurrences back to 3.6.8 for now, thanks, good catch!